build-tools/installers/create-installers.targets

```diff
@@ -410,6 +414,7 @@
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Analysis.Tasks.pdb" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Debugging.Tasks.dll" />
     <_MSBuildFiles Include="$(MSBuildSrcDir)\Xamarin.Android.Build.Debugging.Tasks.pdb" />
+    <_MSBuildFiles Include="@(_LocalizationLanguages->'$(MSBuildSrcDir)\%(Identity)\Xamarin.Android.Build.Debugging.Tasks.resources.dll')" />
```

Comment:
Whoops, it seems like we might want to fix this one in another PR and `Xamarin.Android.Tools.AndroidSdk.resources.dll` if they need to be in 16.9?

Reply:
Thanks, I moved these into https://github.com/xamarin/xamarin-android/pull/5481.